src/components/MapboxMap.tsx

```diff
@@ -9,7 +9,12 @@ import ReactDOM from 'react-dom';
  *
  * @public
  */
-export type PinComponent<T> = (props: { result?: Result<T>, index?: number }) => JSX.Element;
+export type PinComponent<T> = (props: {
+  coordinate: Coordinate,
```

Comment:
Adding a `coordinate` attribute here feels a bit strange to me. It reads as somewhat redundant since `result` is also provided (from which `coordinate` is derived). I definitely understand adding a reference to the mapbox object for various pin effects, etc. 

Could we undo the addition of `coordinate` in this PR? I think that requires some further thought.

Reply:
Yeah I can remove it